api-tests/env/helpdeskcommands_uat_blue.env.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
delete this file, for this first beta release we are not introducing blue green deployment

Reply:
done 👍